ui-v2/app/styles/base/components/pill/layout.scss

```diff
@@ -8,4 +8,5 @@
   padding: 0;
   height: 10px;
   margin-right: 3px;
+  font-size: 0;
```

Comment:
This seems out of place. What's the story here?

Reply:
Ah yeah, I almost didn't put this in here (see the P.S. in the PR description). It was just easier to fit in here, plus this PR is all about 'phrase-editor' amends, which the above fits into.

Basically I was trying to keep the 'Remove' text that is inside the little delete buttons for styling/a11y reasons, it got removed in a previous PR without me realizing and 'Remove' text was then visible. This just rehides it.